contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_us_state_abbreviation.py

```diff
@@ -0,0 +1,145 @@
+import json
+from typing import Optional
+
+import us
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.execution_engine import PandasExecutionEngine
+
```

Comment:
One minor change re: `isort` -- let's remove this line-break.

Reply:
got it!